cargo/lib/dependabot/cargo/file_updater/lockfile_updater.rb

```diff
@@ -269,6 +308,7 @@ def pin_version(content)
           TomlRB.dump(parsed_manifest)
         end
 
+        sig { params(parsed_manifest: T::Hash[T.untyped, T.untyped]).void }
```

Comment:
Is there a better type we could use here? Maybe this:


```suggestion
        sig { params(parsed_manifest: T::Hash[String, String]).void }
```

Reply:
Thank you for the review! Since TOML values are not always strings (they can be numbers or hashes, etc.), Sorbet raised a type error, so I used T::Hash[String, T.untyped] instead.

https://github.com/dependabot/dependabot-core/pull/12523/commits/c7f251157d081c3906f949b988cdbd8320060253